DNN Platform/Modules/DnnExportImport/Components/Common/CompressionUtil.cs

```diff
@@ -158,7 +158,7 @@ public static bool AddFileToArchive(string file, string archivePath, int folderO
 
         #region Private Methods
 
-        private static bool AddFileToArchive(ZipArchive archive, string file, int folderOffset, string folder = null)
+        public static bool AddFileToArchive(ZipArchive archive, string file, int folderOffset, string folder = null)
```

Comment:
At least this could be been moved outsize private methods region

Reply:
I agree, missed that. However I would even prefer to remove all regions and replace that by stylecop rules for order of methods...